package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "magic-script-typings",
-  "version": "1.5.1",
+  "version": "1.6.0",
```

Comment:
This shouldn't be 1.6.0, it's more like 1.5.2.
1.6.0 will contain all API updates that match Lumin Runtime

Reply:
It's now `1.6.1` since master that I rebased on if `1.6.0`